cmd/kaspawallet/libkaspawallet/bip32/version.go

```diff
@@ -38,12 +38,72 @@ var KaspaMainnetPublic = [4]byte{
 	0x2e,
 }
 
+// KaspaTestnetPrivate is the version that is used for
+// kaspa testnet bip32 public extended keys.
+var KaspaTestnetPrivate = [4]byte{
+	0x03,
```

Comment:
Please use either decimal or hex for these values, but not a mixture. (This also refers to the fact that the previous versions were all hexa, and the new ones are mostly decimal).

Also, maybe write in the comment what this encodes to? (a.k.a. kpriv/tpub/etc)